EncoreTickets.SDK/Api/Utilities/RestClientBuilder/ApiRestClientBuilder.cs

```diff
@@ -88,10 +90,14 @@ private static string GetBuildNumber()
             foreach (var property in properties)
             {
                 var propertyValue = property.GetValue(queryObject, null);
-                if (propertyValue != null)
+                if (propertyValue == null)
```

Comment:
Maybe we can move the foreach body into a separate method and then get the result using "properties.Select(...)"?